<launch>
	<node name="urg_node" pkg="urg_node" type="urg_node" />

	<node name="my_vfh_node" pkg="ukftest" type="my_vfh_node" />

	<node name="serial_to_uav_node" pkg="serial_to_uav" type="serial_to_uav_node"  />
	<param name="is_write_on" value="true" />
	


	<node name="uav_b_laser_controller_node" pkg="ukftest" type="uav_b_laser_controller_node"  output="screen"/>
	<!-- PID parameters -->

	<param name="Kp_avoid_vel" value="2.0" />
	<param name="Ki_avoid_vel" value="0.0" />
	<param name="Kd_avoid_vel" value="0.3" />

	<param name="pid_gain" value="1.0" />
	<param name="controlLimit" value="5.0" />
	

</launch>


